qna/DE-custom-datacenter-hosting.md

```diff
@@ -0,0 +1,65 @@
+## Question
+
+Wir arbeiten lediglich mit Eingabegeräten /Terminals, die keine Offlinefunktionalität bieten und nur mit einer Verbindung zum Rechenzentrum Aufzeichnungen anstoßen können. Die Aufzeichnungen erfolgen ausschließlich auf den Servern im Rechenzentrum (Cloud). Damit ist das Rechenzentrum das Operational Environment. Muß ich in meinem Rechenzentrum eine TSE betreiben?
+
+## Metadata tags
+
+lang-en, market-de, middleware, PosCreator, PosDealer, Consultant
+
+## Answer
+
+Die fiskaltrust Lösung kann bei Bedarf unter bestimmten Rahmenbedingungen auch im eigenen Rechenzentrum betrieben werden.
+
+### Voraussetzungen
+
+- Vertragslaufzeit: 3 Jahre 
+- Sie stellen einen dedizierten Kubernetes Cluster je Rechenzentrum bzw. beauftragen einen Dritten mit deren Bereitstellung
+- fiskaltrust übernimmt & konfiguriert den bereitgestellten Cluster
+- fiskaltrust übernimmt das Management des Clusters inkl. SLA
+- fiskaltrust stellt individuelle Produktbundles pro Cashboxen zur Verfügung
+
```

Comment:
Stimmt das?  "
- Die SMAERS wird je CashBox durch fiskaltrust im Rechenzentrum des Kassenherstellers betrieben
- Der CSP (aktuell ist eine HSM/Hardware Security Module erforderlich) wird entweder im Rechenzentrum des Kassenherstellers oder in einem externen Rechenzentrum betrieben"

Reply:
@christian-rogobete @steininger is this correct?

Reply:
Ich verstehe nicht genau, was da eine mit dem anderen zu tun hat. @AxelKutschera könntest du es bitte detaillierter beschreiben? Ansonsten finde ich die Antwort ausreichend. Tiefere Details können im Einzelgespräch geklärt werden.

Reply:
Die Hauptfragen dahinter sind: 
- In welchem Rechenzentrum wird die HSM für den CSP betrieben? 
- In welchem Rechenzentrum wird die SMAERS betrieben?